news.xml

```diff
@@ -7,7 +7,99 @@
       NixOS 20.09 released
     </title>
     <description>
-      Placeholder for the Release Announcement
+      <a href="https://github.com/NixOS/nixos-artwork/blob/master/releases/20.09-nightingale/nightingale.png">
+        <img class="inline" src="logo/nixos-logo-20.09-nightingale-lores.png" alt="20.09 Nightingale logo" with="100" />
+      </a>
+      <p>
+        Hey everyone, I'm <a href="discourse.nixos.org/u/jonringer">Jonathan Ringer</a>,
+        one of the release managers for 20.09. As promised, the latest stable
+        release is here: NixOS 20.09 “Nightingale”.
+      </p>
+
+      <ul>
+        <li>
+          <a href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09">Release manual</a>
+          <ul>
+            <li><a
+                href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09-highlights">Highlights</a>
+            </li>
+            <li><a href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09-new-services">New
+                Services</a></li>
+            <li><a
+                href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09-incompatibilities">Backward
+                Incompatibilities</a></li>
+            <li><a
+                href="https://nixos.org/manual/nixos/stable/release-notes.html#sec-release-20.09-notable-changes">Other
+                Notable Changes</a></li>
+          </ul>
+        </li>
+        <li><a href="https://nixos.org/manual/nixos/stable/index.html#sec-upgrading">Upgrade Instructions</a></li>
+        <li><a href="https://nixos.org/download.html#section-nixos">Image downloads</a></li>
+      </ul>
+
+      <p>
+        <strong>TODO: @jon - change number of contributors and commits.</strong>
+        The 20.09 release was possible because of work of <strong>400</strong>
+        contributors in <strong>20000</strong> commits. We would especially
+        like to thank our top 10 contributors:
+        <a href="https://github.com/">NameA</a>,
+        <a href="https://github.com/">NameB</a>,
+        ...
+      </p>
+
+      <p>
+        NixOS is already known as
+        <a href="https://repology.org/repositories/statistics/newest">the most up to date distribution</a>
+        and is
+        <a href="https://repology.org/repositories/statistics/total">at the top three by total number of packages</a>.
+      </p>
+
+      <p>
+        This didn't stop us. In last six months:
+      </p>
+
+      <ul>
+        <li>7349 packages were added,</li>
+        <li>8181 packages were removed,</li>
+        <li>14442 packages were updated.</li>
+      </ul>
+
+      <p>
+        Likewise our NixOS module system got bigger and reacher:
```

Comment:
```suggestion
        Likewise, our NixOS module system increased in size:
```

Or something.

Reply:
yea, i saw that as well, I'm going to work on this after finishing the mark broken stuff